### ISSUE TITLE
Token ids reset on server start

### ISSUE THREAD

**Issue by Merudo** [RPTools/maptool#619]
**Describe the bug**
The token ids are changed after server start. This can lead to multiple inconveniences.

**To Reproduce**
1. Create a new map with Dragon token.
2. Type

`[findToken("Dragon")]`

3. Start server.
4. Type

`[findToken("Dragon")]`

5. Notice the two ids don't match up.

**Expected behavior**
The token ids don't change.

**MapTool Info**
- Version: 1.5.2, 1.5.3, 1.5.4 RC1
- Install: New

**Desktop (please complete the following information):**
 - OS: Windows
 - Version 10

**Additional context**
[Token ID changes between saves](http://forums.rptools.net/viewtopic.php?t=16060)
[Token IDs Being Reset on Campaign Load](http://forums.rptools.net/viewtopic.php?f=3&t=28214)
[[FREQ] Static Token ID's](https://forums.rptools.net/viewtopic.php?f=26&t=26649)

**Comment by Merudo** [comment]
PR #620 fixes this issue.

**Comment by source-knights** [comment]
The campaign branch is not for that fix, the token id fix was just needed
for the overall purpose of that branch: to store a campaign as an XML
files, so that each map, token, etc has its own file and can be handled in
git.

When people were complaining that the files change every time they save
without any change (so git would always think it was updated), I found that
token ID bug and implemented the workaround. The cut&paste behaviour is
more or less a feature, I wanted to fix it but based on discussions that
this will be rewritten with upcoming MapTool releases anyway and some guys
wanted to test the specific behaviour cause they liked it I left it as it
is.

If you want to remove the setId method have fun with it ;) I did not invest
any more time cause was told that all this will change with the next
MapTool anyway. So I stopped developing and just use my own fork :)
